### PR TITLE
Issue #KN-50 feat: Angular migration from version 11 to 12

### PR DESCRIPTION
**Pull request by gouravmore**
Implementation video:- https://drive.google.com/file/d/1KxhKdlpYfzm-CV41kcm3dL3v2X7whRyH/view?usp=drivesdk
<table>
<td>
  <a href="https://project-sunbird.atlassian.net/browse/KN-50" title="KN-50" target="_blank">RFCKN-50</a>  PDF Player - Angular migration from version 9 to 12
  </td></table>
  <br />
 

[Issue #KN-50 feat: Angular migration from version 9 to 12](https://github.com/project-sunbird/sunbird-pdf-player/commit/c4a4a0aeba42c222a2dbee6bf2d20774c2d89fb5)

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions:
* Hardware versions:

### Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by HarishGangula — CHANGES_REQUESTED** [review]

File: angular.json [hunk 1]
```diff
@@ -217,7 +223,13 @@
             ],
             "scripts": [
               "node_modules/@project-sunbird/telemetry-sdk/index.js"
-            ]
+            ],
+            "vendorChunk": true,
+            "extractLicenses": false,
+            "buildOptimizer": false,
+            "sourceMap": true,
+            "optimization": false,
```
[the remark this answers is not in the document]
Comment (gouravmore) [reply angular.json:231]: It's got auto updated from migration script of 11 to 12

[the remark this answers is not in the document]
Comment (HarishGangula) [reply angular.json:231]: Check what is the correct value for each and update here about it

[the remark this answers is not in the document]
Comment (gouravmore) [reply angular.json:231]: It is newly added by migration script, correct values are true for all but it is already added in production configuration line https://github.com/project-sunbird/sunbird-pdf-player/pull/83/files#diff-2d2675bb4687601a5c7ccf707455132f90f3516a33716185687e5c41df59ac7dL55 to line to https://github.com/project-sunbird/sunbird-pdf-player/pull/83/files#diff-2d2675bb4687601a5c7ccf707455132f90f3516a33716185687e5c41df59ac7dL63

[the remark this answers is not in the document]
Comment (gouravmore) [reply angular.json:231]: There is no change in existing code

**Pull request merged**